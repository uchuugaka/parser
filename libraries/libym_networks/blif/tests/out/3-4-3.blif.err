"3-4-3.blif" line: 5, column: 10 [WR_MLTDEF02]: a: Defined as both input and output. Previous definition is "3-4-3.blif" line: 4, column: 9.
