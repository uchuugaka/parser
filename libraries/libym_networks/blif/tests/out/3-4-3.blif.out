.model test1
	["3-4-3.blif" line: 3, columns:1-6]
	["3-4-3.blif" line: 3, columns:8-12]
  inputs elem: a
	["3-4-3.blif" line: 4, column: 9]
  inputs elem: b
	["3-4-3.blif" line: 4, column: 11]
  outputs elem: a
	["3-4-3.blif" line: 5, column: 10]
  outputs elem: x
	["3-4-3.blif" line: 6, column: 10]
.names
a
	["3-4-3.blif" line: 7, column: 8]
b
	["3-4-3.blif" line: 7, column: 10]
x
	["3-4-3.blif" line: 7, column: 12]
11 1
.end
	["3-4-3.blif" line: 9, columns:1-4]
normal_exit
